pylint_odoo/checkers/modules_odoo.py

```diff
@@ -409,11 +412,16 @@ def _check_duplicate_xml_record_id(self):
                  add list of errors in self.msg_args
         """
         self.msg_args = []
-        all_xml_ids = []
-        for xml_file in self.filter_files_ext('xml', relpath=False):
-            all_xml_ids.extend(self.get_xml_records(xml_file))
+        xml_records = []
+        for fname, section in self._get_manifest_referenced_files().items():
```

Comment:
OK, I get it. Then the comment should be: `Check duplicated XML-IDs inside of the files of each section treated them separately`